### DISCUSSION TITLE
Images not uploading on first or second attempt from mobile browser

### DISCUSSION THREAD

**Discussion by Str1atum** [immich-app/immich#11548] [Q&A]
Mostly I upload images from mobile Safari with latest iOS versions (iPad or iPhone) but most of the time, the first or second attempt does not work so I just have to retry uploading the same stack of images again for 1-2 times and then it works flawless. There's not error message ever, I just select the images in iOS image picker and hit Add but then the screen disappears and I'm back on the immich site.
Anyone else experiencing this behavior?

**Comment by alextran1502** [comment]
Probably weird Safari mobile api that hasn’t been handled correctly since we have a mobile app. Just curious, why don’t you use the mobile app to upload your assets? Have you also tried different mobile browser?

**Reply by Str1atum** [reply]
Works with Chrome but Chrome is not uploading HEIF but converts to JPEG before.
The app doesn't allow a simple upload of specific images, that's why I use upload via the web site.
